addons/account_peppol/i18n/account_peppol.pot

```diff
@@ -761,12 +765,12 @@ msgstr ""
 
 #. module: account_peppol
 #: model:ir.model.fields.selection,name:account_peppol.selection__account_move__peppol_move_state__to_send
-msgid "Queued"
+msgid "Pending"
```

Comment:
i think that if you export the pot from the interface the change in line 617 and 768 will be put together :smile: Can you export it instead of just replacing the terms, specially because we keep the pot in alphabetical order :smile: